packages/dom-helper/lib/prop.js

```diff
@@ -2,6 +2,9 @@ export function isAttrRemovalValue(value) {
   return value === null || value === undefined;
 }
 
+var getOwnPropertyDescriptor = Object.getOwnPropertyDescriptor;
```

Comment:
Thinking more about this: we have IE8 in CI so if tests are green (and we are testing this path), we are good.